lib/node_modules/@stdlib/repl/lib/commands/add_theme.js

```diff
@@ -0,0 +1,74 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable no-underscore-dangle */
+
+'use strict';
+
+// MODULES //
+
+var format = require( '@stdlib/string/format' );
+var log = require( './../log.js' );
+
+
+// MAIN //
+
+/**
+* Returns a callback to be invoked upon calling the `addTheme` command.
+*
+* @private
+* @param {REPL} repl - REPL instance
+* @returns {Function} callback
+*/
+function command( repl ) {
+	return onCommand;
+
+	/**
+	* Adds a color theme to the syntax-highlighter.
+	*
+	* @private
+	* @param {string} [name] - theme name
+	* @param {Object} [theme] - theme object
+	* @returns {void}
+	*/
+	function onCommand() {
+		var theme;
+		var nargs;
+		var name;
+
+		nargs = arguments.length;
+		if ( nargs < 2 ) {
```

Comment:
You don't need the temporary `nargs` variable here.

Reply:
Also, why are we using `arguments` at all? You can just do `onCommand( name, theme )`.

Reply:
In which case, the `nargs < 2` check and error message are unnecessary, as L62 should take care of validation.